packages/amplify_auth_cognito_plugin/amplify_auth_cognito_android/example/README.md

```diff
@@ -0,0 +1,3 @@
+# amplify_auth_cognito_android_example
```

Comment:
Do the platform packages need examples still if all the tests are in the main pkg's example?